en/docs/observe/api-manager-analytics/overview-of-api-analytics.md

```diff
@@ -1,221 +1,45 @@
-
-</tr>
-<tr>
-<th>Information Provided</th>
-<td>Users can download a PDF report of the API usage statistics for the selected month. Later this report can be used for billing purposes. 
-<div class="admonition note">
-<p class="admonition-title">Note</p>
-<p>You can customize the format and structure of this dashboard as explained in the <a href="{{base_path}}/observe/api-manager-analytics/monthly-api-usage-report/#customizing-the-report">Monthly API Usage Report</a> section.</p>
-
-</td>
-</tr>
-
-<tr>
-<th>
-Targeted Use case
-</th>
-<td>This can be used to accomplish the API monetization use cases.
-</td>
-</tr>
-<tr>
-<th>
-Permission Needed
-</th>
-<td>Currently, this is restricted to users with the <code>admin</code> role only.
-</td>
-</tr>
-<tr>
-
-<td colspan="2">For more details, see the <a href="{{base_path}}/observe/api-manager-analytics/monthly-api-usage-report">Monthly API Usage Report</a> section.
-</td>
-</tr>
-</table>
-
-## Customizing the Analytics Dashboards
-
-Apart from the latter mentioned default dashboards, the users can create custom dashboards based on their specific requirements as explained in **[Customizing Analytics Dashboards]({{base_path}}/observe/api-manager-analytics/customizing-analytics-dashboards)**.
+![APIM Analytics Overview Page]({{base_path}}/assets/img/observe/apim-analytics-overview-page.png)
+
+APIs are widely used in Business Integrations and are becoming a key part of driving an organization's business 
+strategy. Ensuring the proper functioning of the APIs, getting feedback on the performance and deriving business 
+insights from APIs are becoming equally important.
+
+With the API Manager 4.0.0 release API Analytics was completely revamped and offered as a cloud-based analytics solution
+ to provide users with an unparallel analytics experience addressing many customer concerns we received for earlier 
+ releases.
+
+## Highlights
+### Hassle Free Setting Up
+One of the major concerns customers had earlier was the effort taken up to setup the analytics environment. This 
+involves setting up a minimum of two node deployment with tuned parameters which is a considerable effort and utilizes 
+considerable amount of resources. 
+With all new API Manager 4.0.0 cloud-based analytics model all that you need to do is,
+ - [Register for API Manager Analytics]({{base_path}}/observe/api-manager-analytics/configure-analytics/register-for-analytics)
+ - [Configure Synapse Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-synapse-gateway) or [Choreo Connect Gateway]({{base_path}}/observe/api-manager-analytics/configure-analytics/configure-microgateway)
+
+These are the only two steps that is needed to setup analytics with the new solution
+
+### Modern Responsive UIs
+Earlier versions on API Manager analytics relied on home drown dashbaording solution for which We received the
```

Comment:
Let's remove this as well.

Reply:
Fixed with ceadce23591c5187ed8bf6e0e424b49e64832bcf